### PR TITLE
Fix ESBJAVA-5077

### PR DESCRIPTION
**Pull request by bsenduran**
Fix Prepared statement not closed properly, diff by @sajithshn

### CODE REVIEW

File: modules/core/src/main/java/org/apache/synapse/mediators/db/DBLookupMediator.java [hunk 1]
```diff
@@ -112,6 +113,11 @@ protected void processStatement(Statement stmnt, MessageContext msgCtx) {
                     rs.close();
                 } catch (SQLException e) {}
             }
+            if (ps != null) {
+                try {
+                    ps.close();
+                } catch (SQLException ignore) {}
```
Comment (madhawa-gunasekara) [inline modules/core/src/main/java/org/apache/synapse/mediators/db/DBLookupMediator.java]: shall we print a warning message, without ignoring the exception

Comment (bsenduran) [inline modules/core/src/main/java/org/apache/synapse/mediators/db/DBLookupMediator.java]: @madhawa-gunasekara Good point, but all the other similar close statements are ignoring the exception, hence ignored to have a consistence. IMO, if we added a warning we may need to add to the other close statements as well.

Comment (madhawa-gunasekara) [inline modules/core/src/main/java/org/apache/synapse/mediators/db/DBLookupMediator.java]: yes, let's add to other close statements as well.

Comment (bsenduran) [inline modules/core/src/main/java/org/apache/synapse/mediators/db/DBLookupMediator.java]: sure, we will keep track on this and add warning logs soon

**Pull request merged**